src/transaction/log_manager.c

```diff
@@ -14138,48 +14138,61 @@ cdc_get_start_point_from_file (THREAD_ENTRY * thread_p, int arv_num, LOG_LSA * r
   else
     {
       LOG_ARCHIVE_CS_ENTER (thread_p);
-      aligned_hdr_pgbuf = PTR_ALIGN (hdr_pgbuf, MAX_ALIGNMENT);
 
-      hdr_pgptr = (LOG_PAGE *) aligned_hdr_pgbuf;
+      if (log_Gl.archive.vdes != NULL_VOLDES && log_Gl.archive.hdr.arv_num == arv_num)
+	{
+	  /* if target archive log volume is currenty mounted, then use that */
+	  process_lsa.pageid = log_Gl.archive.hdr.fpageid;
+	  process_lsa.offset = 0;
+	}
+      else
+	{
+	  aligned_hdr_pgbuf = PTR_ALIGN (hdr_pgbuf, MAX_ALIGNMENT);
 
-      fileio_make_log_archive_name (arv_name, log_Archive_path, log_Prefix, arv_num);
+	  hdr_pgptr = (LOG_PAGE *) aligned_hdr_pgbuf;
 
-      if (fileio_is_volume_exist (arv_name) == true)
-	{
-	  vdes = fileio_mount (thread_p, log_Db_fullname, arv_name, LOG_DBLOG_ARCHIVE_VOLID, false, false);
-	  if (vdes != NULL_VOLDES)
+	  fileio_make_log_archive_name (arv_name, log_Archive_path, log_Prefix, arv_num);
+
+	  if (fileio_is_volume_exist (arv_name) == true)
```

Comment:
It would be nice if handling 'else' case too. In the perspective of code, the process_lsa was not initialized and it is passed to the logpb_fetch_page().

Reply:
11.1 patch please.

Reply:
Yes, you are right. 
Attempts were made only to check against archive log volumes that already exist through the information stored in log_Gl. 

However, I'll add an error along with assert() as it can cause an error when that archive volume is deleted.